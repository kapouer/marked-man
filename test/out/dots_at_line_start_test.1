DOTS_AT_LINE_START_TEST()                            DOTS_AT_LINE_START_TEST()

NNAAMMEE
       ddoottss__aatt__lliinnee__ssttaarrtt__tteesstt

       There's a weird issue where dots at the beginning of a line
       generate troff warnings due to escaping.

       .. let's see what happens.

                                 January 1979        DOTS_AT_LINE_START_TEST()
