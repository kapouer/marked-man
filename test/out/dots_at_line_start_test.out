[4mDOTS_AT_LINE_START_TEST[24m(1)  General Commands Manual [4mDOTS_AT_LINE_START_TEST[24m(1)

[1mNAME[0m
       [1mdots_at_line_start_test[0m

       Thereâ€™s a weird issue where dots at the beginning of a line
       generate troff warnings due to escaping.

       .. letâ€™s see what happens.

                                 January 1979       [4mDOTS_AT_LINE_START_TEST[24m(1)
